src/AccessibilityInsights/MainWindow.xaml.cs

```diff
@@ -159,6 +159,9 @@ public MainWindow()
 
             SetHighContrastTheme();
 
+            ///in case we need to do any debugging with elevated app
+            SupportDebugging();
```

Comment:
Why was this moved? Curious

Reply:
I sometimes hit errors in `PopulateConfigurations` that occurred too early. This way we can step through PopulateConfigurations